state.c

```diff
@@ -444,6 +444,8 @@ int pv_state_validate(struct pv_state *s)
 
 static int pv_state_mount_bsp_volumes(struct pv_state *s)
 {
+	pv_disk_mount_swap(s);
```

Comment:
I'd move this out of here as it has nothing to do with mounting bsp volumes. You could do it before the call to pv_state_mount_bsp_volumes in pv_state_start.

@asac  would it be convenient to fail and return -1 if disks are not mounted, so we rollback in that case?